src/authservice/chart/values.yaml

```diff
@@ -114,20 +29,7 @@ tolerations: []
 
 affinity: {}
 
-# -- Log level for the deployment
-config:
-  logLevel: trace
-
 # -- Label to determine what workloads (pods/deployments) should be protected by authservice.
 selector:
   key: protect
   value: keycloak
```

Comment:
We can likely drop these values from here at this point AND remove the corresponding `remotePodLabels` from the [Package CR](https://github.com/defenseunicorns/uds-core/blob/authservice-pepr/src/authservice/chart/templates/uds-package.yaml#L21-L28).